### DISCUSSION TITLE
How do you backup localhost (docker host) if localhost is the docker container?

### DISCUSSION THREAD

**Discussion by bdwilson** [tiredofit/docker-backuppc#19] [Q&A]
I'm moving from bare metal to docker for backuppc and struggling with how to backup the docker host (localhost).  Is it best to do this via SSH?  Wondering from CPU and bandwidth perspective how is the best way to accomplish this.

**Comment by tiredofit** [comment]
You could perform a bind mount of your `/` to `/rootfs` and backup that way, however I feel SSH would be the better host agnostic approach to allow for moving the container and data easier.

**Reply by bdwilson** [reply]
Thank you for the quick response. I couldn’t find any guidance given it’s a non-standard ask for bare metal deployments. I’ll use ssh and see how it goes. Thanks!
